packages/web-kit/src/console-auth.css

```
@@ -0,0 +1,700 @@
+  --ink-4: #86868b;
+  --line: #d2d2d7;
+  --line-soft: #e8e8ed;
+  --paper: #ffffff;
+  --paper-2: #fbfbfd;
+  --paper-3: #f5f5f7;
+  --ok: #2f9e63;
+  --info: #3b82f6;
+  --warn: #c98a14;
+  --err: #d8483a;
+
+  --font-sans: "Inter", "Noto Sans JP", -apple-system, BlinkMacSystemFont, system-ui, sans-serif;
```

Stylelint (17.13.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)